docs/src/kits/react/create-react-app.md

```diff
@@ -289,8 +289,29 @@ yarn deploy
 You should now have a React Application on the Permaweb! Great Job!
 :::
 
-::: error
-If you receive this error `Not enough funds to send data`, you have to fund some AR into your wallet, and then try to deploy it again.
+::: info ERROR
+If you receive this error `Not enough funds to send data`, you have to fund some AR into your Bundlr wallet, and then try to deploy it again, run
+:::
+
+<CodeGroup>
+  <CodeGroupItem title="NPM">
+  
+```console:no-line-numbers
+bundlr fund 1479016 -h https://node1.bundlr.network -w wallet.json -c arweave
```

Comment:
just tell the user little bit about the smallest unit of the AR

Reply:
Done! thanks Saif

Reply:
Awesome @VinceJuliano